telemetry/jetbrains/src/main/kotlin/software/aws/toolkits/telemetry/generator/GenerationFunctions.kt

```diff
@@ -0,0 +1,186 @@
+package software.aws.toolkits.telemetry.generator
```

Comment:
why move this to a separate file?

Reply:
I'm fine with merging it back into one file. The split made sense, but after more refactoring it doesn't.